bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@satishd I dont think so. We are not exporting any component spec jsons. We should be okay since the fieldName has not changed.